CIP-0088/README.md

```diff
@@ -0,0 +1,867 @@
+A pointer to a "Beacon" or Reference Token that may utilize an inline datum format to inject information into smart
+contracts and a Datum Format which would enable marketplaces to pay royalties to a smart contract address allowing for
+greater royalty flexibility. Making further argument for the presence or lack thereof of these data points in CIP-27 is
+outside the scope of this CIP.
+
+**Example (CBOR):**
+
+```cbor 
+{
+  27: {
+    0: "1.0.0",
+    1: {
+      1: "0.05",
+      2: [
+        "addr_test1qqp7uedmne8vjzue66hknx87jspg56qhkm4gp6ahyw7kaahevmtcux",
+        "lpy25nqhaljc70094vfu8q4knqyv6668cvwhsq64gt89"
+      ],
+      3: [
+        "d894897411707efa755a76deb66d26dfd50593f2e70863e1661e98a0",
+        "7370616365636f696e73"
+      ],
+      4: {
+        0,
+        ""
+      }
+    }
+  }
+}
+```
+
+##### CIP-48: Metadata References Standard
+
+CIP-48 has been proposed to extend on-chain metadata formats to support "references" which could be:
+
+1. Shared/Repeated pieces of content
+2. Pointers or References to other on-chain information
+
+CIP-88 could help support the definition and standardization of the reference definitions at a top-level, allowing
+references declared within individual token metadata to be easily identified and replaced.
+
+***CIP-48: Top-Level Object Fields***
+
+| index | name              | type   | required | notes                                                          |
+|-------|-------------------|--------|----------|----------------------------------------------------------------|
+| 0     | Version           | String | No       | Default "1.0.0", which version of this specification is in use |
+| 1     | Reference Details | Object | No       | An object detailing CIP-48 references in use                   |
+
+***Reference Details Fields***
+
+| index | name       | type  | required | notes                  |
+|-------|------------|-------|----------|------------------------|
+| 1     | References | Array | No       | An array of References |
+
+***Reference Fields***
+
+| index | name    | type   | required | notes                                                                                                                 |
+|-------|---------|--------|----------|-----------------------------------------------------------------------------------------------------------------------|
+| 0     | Name    | String | Yes      | A case sensitive path identifier for this reference                                                                   |
+| 1     | Type    | Enum   | Yes      | An enum of accepted types of references which may include direct payloads or pointers to other sources of information |
+| 2     | Payload | Object | Yes      | A "Reference Payload" object containing the information to be substituted in place of the reference                   |
+
+**TODO: Expand Support for CIP-48**
```

Comment:
@Jack-0 see here for addressing the potential for supporting CIP-48 reference data via CIP-88 :)

Reply:
Also asked here (https://github.com/cardano-foundation/CIPs/pull/249#discussion_r1150384616) if this approach now described in CIP-0088 now leaves this proposal deprecated:

- https://github.com/cardano-foundation/CIPs/pull/343